torch/sparse/__init__.py

```diff
@@ -218,7 +228,9 @@ def sum(input: Tensor, dim: DimOrDims = None,
             return torch._sparse_sum(input, dtype=dtype)
 
 
-softmax = _add_docstr(_sparse._sparse_softmax, r"""
```

Comment:
Nit: I know it might be annoying, but ideally we don't land code formatting changes that touch unrelated pieces of code. It inflates the number of changes and changes the blame, which makes it harder to track them on the github UI.